frontend/msSettingsManager.pas

```diff
@@ -480,6 +482,37 @@ procedure TSettingsManager.BuildFromPluginsItemClick(Sender: TObject);
   slSelection.Free;
 end;
 
+procedure TSettingsManager.SelectSimilarNodesItemClick(Sender: TObject);
+var
+  i, index: Integer;
+  tvNode: TTreeNode;
```

Comment:
Why "tvNode", couldn't we just call it "node"?  I don't know of any kind of "Node" type in the project other than a TTreeNode, so specifying this is a "TreeView" node doesn't seem fully necessary.

Also, per hungarian notation I've been using, prepending "tv" to a variable means it's a TTreeView variable.  Let's just go with node.